ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/sorter/AbstractPendingTransactionsSorter.java

```diff
@@ -97,6 +98,8 @@ public abstract class AbstractPendingTransactionsSorter implements PendingTransa
   protected final TransactionPoolReplacementHandler transactionReplacementHandler;
   protected final Supplier<BlockHeader> chainHeadHeaderSupplier;
 
+  private final BlobCache blobCache;
```

Comment:
just a note to say that adding the blob cache to the `legacy` txpool is a nice to have, but we do not recommend to use it for EIP4844